docs/features/sessions/pause.md

```diff
@@ -41,3 +41,29 @@ This is so that you don't lose your work!
 When you have added credits, just resume the session and carry on with your work. The left navigation bar always shows credits remaining.
 
 ![](/images/sessions/autopausing.gif)
+
+## Pause a Session Using a Github Action
+
+When you are done with your work day, you can setup Github Action to pause your session.  
+
+Below is scheduled-SessionPause.yml:
+
+```
+on:
+  schedule:
+    # Runs "at 8:25pm" everyday
+    - cron: '25 8 * * *'
+jobs:
+  gridai-actions:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: gridai-actions/gridai-login@v0
+        with:
+          gridai-username: ${{ secrets.GRIDAI_USERNAME }} 
+          gridai-key: ${{ secrets.GRIDAI_KEY }}
+      - run: |
+          grid session pause ${{ secrets.SESSION_NAME }} 
+  ```
+
+For example look at the [Github repository](https://github.com/sunitaprakash/grid-session-env).
```

Comment:
Isn't this the exact example taken from your repo? If so why not just link to your repo instead of putting the code in line so that the community is always directed to the most updated example?

Reply:
removed the example just pointed it to the Github repo